captum/attr/_utils/gradient.py

```diff
@@ -62,8 +67,11 @@ def undo_gradient_requirements(inputs, grad_required):
 
 
 def compute_gradients(
-    forward_fn, inputs, target_ind=None, additional_forward_args=None
-):
+    forward_fn: Module,
+    inputs: TensorOrTupleOfTensors,
+    target_ind: Optional[TensorOrTupleOfTensors] = None,
```

Comment:
This type hint should match target, e.g. Optional[Union[int, Tuple[int, ...], Tensor, List[Tuple[int, ...]]]]